kasa/smartdimmer.py

```diff
@@ -1,11 +1,35 @@
 """Module for dimmers (currently only HS220)."""
+from enum import Enum
 from typing import Any, Dict, Optional
 
 from kasa.modules import AmbientLight, Motion
 from kasa.smartdevice import DeviceType, SmartDeviceException, requires_update
 from kasa.smartplug import SmartPlug
 
 
+class ButtonAction(Enum):
```

Comment:
Please rebase on top of the current master, these unrelated changes are already there.